server/server.js

```diff
@@ -11,13 +11,34 @@ app.use(bodyParser.urlencoded({extended: true}));
 // Declare static files
 app.use(express.static(__dirname + '/../client/build'));
 
-app.get('/hello', function(req, res) {
-  res.send('hello');
-});
+// ROUTES --------------------------------------------------
+
+// Should authenticate a user, creating a session token and assigning it
+// to this user.
+app.post('/login', function(req, res) {});
```

Comment:
Should we have a get route for `/login`?

Reply:
Not an issue right now. Will think in the future.

Reply:
I think GET is appropriate